roles/configure_cgroups/tasks/main.yml

```diff
@@ -16,6 +16,13 @@
     sp_cg_set_memsw: '{{ "1" if cgroup_memsw_file.stat.exists else "0" }}'
     cacheable: true
 
+- name: Setting numa_overflow fact
+  set_fact:
+    sp_cg_set_numa_overflow: >-
+      {{ "1" if "storpool_server" in group_names 
```

Comment:
The `storpool_server` group is always defined. The group being defined doesn't indicate that a given host is meant to run the `storpool_server` service. Check this paragraph for more info on how to determine if a host is part of a group: https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_vars_facts.html#information-about-ansible-magic-variables